cirq-ionq/setup.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+import os
+from setuptools import find_packages, setup
+
+# This reads the __version__ variable from cirq/_version.py
+__version__ = ''
+exec(open('cirq_ionq/_version.py').read())
+
+name = 'cirq-ionq'
+
+description = (
+    'A framework for creating, editing, and invoking '
```

Comment:
should this be set to describe that this is the stuff to connect to ionq?

Reply:
yes!